packages/itwinui-css/backstop/tests/divider.html

```diff
@@ -0,0 +1,65 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>Divider | iTwinUI</title>
+
+    <script type="module">
+      import "./assets/theme.js";
+    </script>
+    <style>
+      @import url("./assets/demo.css") layer(demo);
+      @import url('@itwin/itwinui-variables') layer(variables);
+      @import url("@itwin/itwinui-css/css/all.css") layer(itwinui);
+
+      section[id^='demo'] {
+        display: flex;
+        gap: var(--iui-size-xs);
+        width: auto;
+        padding: var(--iui-size-xs);
+      }
+
+      #demo-horizontal,
+      #demo-vertical {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        flex-grow: 1;
+        aspect-ratio: 1/1;
+        border: 1px solid var(--iui-color-border);
+        padding: var(--iui-size-xs);
+      }
+    </style>
+  </head>
+  <body class="iui-root">
+    <theme-button></theme-button>
+    <h1>Divider</h1>
+    <hr />
+
+    <section id="demo-default">
+
+      <div id="demo-horizontal">
+        <hr class="iui-divider" />
```

Comment:
I think Divider doesn't need its own html page. We don't have a page for Flex.

We can still show it in storybook. https://itwin.github.io/iTwinUI/react/?path=/story/utilities-flex--basic

Reply:
Ok, I've removed the demo page, scenario, and test images for divider.